robin_stringer/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "robin_stringer",
+  "version": "0.1.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rastringer/mongo_backed_rest_api.git"
+  },
+  "author": "ra1stringer@gmail.com",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/rastringer/mongo_backed_rest_api/issues"
+  },
+  "homepage": "https://github.com/rastringer/mongo_backed_rest_api#readme",
+  "devDependencies": {
+    "body-parser": "^1.14.1",
```

Comment:
body-parser, express, mongoose are not a devDependency, but a dependency for the app. Mongo is a global install and you shouldn't need to add it in your package.json.

Mocha and chai would be your devDependencies.